### PR TITLE
Add Minikube quickstart guide

### PR DESCRIPTION
**Pull request by Minutis**

### CODE REVIEW

File: quickstart/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+### Enable Minikube `ingress` addon
+```
+minikube addons enable ingress
+```
+
+### Apply quickstart `.yml` configuration
+Assuming you cloned Lighter repo to `~/git/lighter`:
+1. Create main resources
+```
+minikube kubectl -- apply -f git/lighter/quickstart/lighter.yml
```
Comment (pdambrauskas) [inline quickstart/README.md:20]: maybe remove `git/lighter` prefix from the command? In that case user will be able to `cd  whatever_dir/lighter` and just copy/paste these commands

`minikube kubectl -- apply -f quickstart/lighter.yml`

File: quickstart/lighter.yml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            ports:
+              -   containerPort: 8080
+            env:
+              -   name: LIGHTER_KUBERNETES_ENABLED
+                  value: "true"
+              -   name: LIGHTER_MAX_RUNNING_JOBS
+                  value: "15"
+              -   name: LIGHTER_KUBERNETES_SERVICE_ACCOUNT
+                  value: lighter
+      serviceAccountName: lighter
```
Comment (pdambrauskas) [inline quickstart/lighter.yml:95]: new line missing

**Comment by pdambrauskas** [comment]
Could you remove yaml snippets from k8s documentation and add add links to quickstart yaml files, so that we have a single place where we need to maintain k8s yaml examples?

**Comment by pdambrauskas** [comment]
Also we should add a link to Quick Start guide in the main Readme.md file.

**Comment by Minutis** [comment]
> Could you remove yaml snippets from k8s documentation and add add links to quickstart yaml files, so that we have a single place where we need to maintain k8s yaml examples?

> Also we should add a link to Quick Start guide in the main Readme.md file.

Good points - made adjustments.

File: quickstart/lighter/lighter-ingress.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+kind: Middleware
+metadata:
+  name: lighter-custom-headers
+  namespace: spark
+spec:
+  headers:
+    customRequestHeaders:
+      X-Forwarded-Prefix: /lighter
+      X-Forwarded-Proto: https
+      X-Forwarded-Port: "443"
```
Comment (pdambrauskas) [inline quickstart/lighter/lighter-ingress.yml:28]: missing new line

**Review by pdambrauskas — APPROVED** [review]

**Pull request merged**